tcms/core/tests/test_sendtestemail.py

```diff
@@ -9,7 +12,11 @@ class TestSendTestEmail(TestCase):
     @override_settings(EMAIL_HOST='bogus.nonexistent')
```

Comment:
During testing the `EMAIL_BACKEND` is `django.core.mail.backends.locmem.EmailBackend` instead of SMTP.